Makefile

```diff
@@ -5,7 +5,8 @@ build:
 
 deps:
 	git submodule update --init --recursive
-	opam install --deps-only -y .
+	opam install --deps-only -y . vendor/*/
+	dune external-lib-deps --missing @@default
```

Comment:
There should be no missing deps after we call `make deps`, I'm not sure we need this last line.
Also, this new `make deps` does not behave the same way the CI does

Reply:
The added line doesn't install new deps, it's just a diagnosis to help solve dependencies missing in OPAM.
I'll take a stab at having the CI do the same thing.

Reply:
I understood it's just a check, I'm saying I'm not sure this should be part of the normal installation workflow for the user (as described in `README.md`), as the OPAM files should always be correct in the `master` branch.

Reply:
Last line removed for now, since the CI wouldn't be calling this code path anymore.